### PR TITLE
Questions

### PR DESCRIPTION
**Pull request by brokenheart1712**

### CODE REVIEW

File: app/views/admin/questions/_question_form.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<%= form_for [:admin, @question] do |f| %>
+  <%= render 'shared/error_messages', object: f.object %>
```
Comment (PhamVanTung) [inline app/views/admin/questions/_question_form.html.erb]: convert ' to "

**Comment by nguyenngoctruong** [comment]
ok

File: app/controllers/admin/questions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  end
+
+  def destroy
+    @question = question.find params[:id]
+    if @question.destroy
+      flash[:success] = t :success
+    else
+      flash[:danger] = t :fail
+    end
+      redirect_to admin_questions_path
```
Comment (dieunb) [inline app/controllers/admin/questions_controller.rb]: indent

File: app/controllers/admin/questions_controller.rb [hunk 3]
```diff
@@ -0,0 +1,55 @@
+class Admin::QuestionsController < ApplicationController
+  def index
+    @questions = Question.paginate page: params[:page],
+    per_page: Settings.page_size
```
Comment (chuanhtuan) [inline app/controllers/admin/questions_controller.rb]: indent

File: app/helpers/application_helper.rb [hunk 4]
```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  def link_to_add_fields name, f, association
+    new_object = f.object.class.reflect_on_association(association).klass.new
+    fields = f.fields_for(association, new_object,
+    child_index: "new_#{association}") do |builder|
```
Comment (chuanhtuan) [inline app/helpers/application_helper.rb]: indent

**Pull request merged**